internal/database/orgs.go

```diff
@@ -308,3 +309,54 @@ func (o *orgStore) Delete(ctx context.Context, id int32) (err error) {
 
 	return nil
 }
+
+func (o *orgStore) HardDelete(ctx context.Context, id int32) (err error) {
+	// Check if the org exists even if it has been previously soft deleted
+	orgs, err := o.getBySQL(ctx, "WHERE id=$1 LIMIT 1", id)
+	if err != nil {
+		return err
+	}
```

Comment:
99% sure that if there is no record exists, `getBySQL` returns error `sql.ErrNoRows`, so what you actually want would be:
```suggestion
	orgs, err := o.getBySQL(ctx, "WHERE id=$1 LIMIT 1", id)
	if err != nil {
		if err == sql.ErrNoRows {
			return nil
		}
		return err
	}
```

Ignore this if not the case.

Reply:
`o. getBySQL`  returns an empty array when no record is found, so I will skip this suggestion.